### PR TITLE
Add admins to carbonplan hub

### LINKED ISSUE #566: Add JupyterHub admin users to CarbonPlan hub
# Summary

I'd like to add 3 folks to the list of JupyterHub admins on the CarbonPlan hub. Those are:

- @jhamman (me!)
- @orianac
- @tcchiao 

# User Stories

- We need the ability for a few folks on our core team to administer JupyterHub sessions.

# Acceptance criteria

- All users listed above should be able to access the admin dashboard on the hub

# Important information

# Tasks to complete

- [ ] add users to list of hub admins

### PR DESCRIPTION
**Pull request by choldgraf**
I tried to mimic how this is done on the OpenScapes hub

closes #566

### CODE REVIEW

**Comment by yuvipanda** [comment]
Let's do it like https://github.com/2i2c-org/pilot-hubs/blob/151faa89e92d7ca596a31bbe9e58e16de758a2f7/config/hubs/2i2c.cluster.yaml#L50 instead? That'll mean only the listed users can log in (right now anyone with GitHub can login lol), and they can add more users in the admin panel when they need to log in.

**Comment by yuvipanda** [comment]
Also https://github.com/2i2c-org/pilot-hubs/pull/558 is needed to deploy to the new carbonplan hub

**Comment by choldgraf** [comment]
ok I think this is what you mean? Do we also need to do this in the openscapes hub? https://github.com/2i2c-org/pilot-hubs/blob/151faa89e92d7ca596a31bbe9e58e16de758a2f7/config/hubs/openscapes.cluster.yaml

File: config/hubs/carbonplan.cluster.yaml [hunk 1]
```diff
@@ -151,6 +151,17 @@ hubs:
             readinessProbe:
               enabled: false
             nodeSelector: {}
+            config:
+              Authenticator:
+                allowed_users: &staging_users
```
Comment (yuvipanda) [inline config/hubs/carbonplan.cluster.yaml:156]:
```suggestion
                allowed_users: &users
```

File: config/hubs/carbonplan.cluster.yaml [hunk 2]
```diff
@@ -151,6 +151,17 @@ hubs:
...
+                  - yuvipanda
+                  - choldgraf
+                  - GeorgianaElena
+                  - damianavila
+                  - sgibson91
+                  - jhamman
+                admin_users: *staging_users
```
Comment (yuvipanda) [inline config/hubs/carbonplan.cluster.yaml:163]:
```suggestion
                admin_users: *users
```

**Review by yuvipanda — CHANGES_REQUESTED** [review]
Yep! I think we should do this for openscapes (and all our hubs really), since otherwise anyone with a github login can get in.

The * and & are YAML anchors that need to be unique within the document, so they were `<hubname>_users` in 2i2c.clusters.yaml. Not the case here though.

There's docs on adding new users in https://pilot.2i2c.org/en/latest/admin/howto/manage-users.html, but we should have equivalent in the '2i2c engineer handbook'

**Pull request merged**